### PR TITLE
Advanced tool tips

### PR DESCRIPTION
**Pull request by HeyItsMeNobody**

### CODE REVIEW

File: src/main/java/mounderfod/moundertweaks/mixin/client/ItemMixin.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import java.util.List;
+
+@Mixin(Item.class)
+public abstract class ItemMixin {
+    // "Improved" and rewritten Shnupbups code.
+
+    @Inject(method = "appendTooltip(Lnet/minecraft/item/ItemStack;Lnet/minecraft/world/World;Ljava/util/List;Lnet/minecraft/client/item/TooltipContext;)V", at = @At("HEAD"))
```
Comment (BoogieMonster1O1) [inline src/main/java/mounderfod/moundertweaks/mixin/client/ItemMixin.java:26]: You don't need a mixin for this

File: src/main/java/mounderfod/moundertweaks/mixin/client/ItemMixin.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import java.util.List;
+
+@Mixin(Item.class)
+public abstract class ItemMixin {
+    // "Improved" and rewritten Shnupbups code.
```
Comment (BoogieMonster1O1) [inline src/main/java/mounderfod/moundertweaks/mixin/client/ItemMixin.java:24]: Explain

**Review by BoogieMonster1O1 — CHANGES_REQUESTED** [review]
this is already it's own mod

**Comment by redcreeper14385** [comment]
Gonna close this for now, sorry.

**Pull request closed without merging**